packages/web/package.json

```diff
@@ -16,6 +16,7 @@
     "livereload": "^0.7.0",
     "rimraf": "^2.6.2",
     "serve-static": "^1.13.2",
+    "sharp": "^0.23.0",
```

Comment:
We need to make sure that's the same version as the other packages. Sharp can't have multiple versions as it uses a dynamically linked library

Reply:
Made sure all versions of sharp are 0.22.1 👍